src/query-builder/delete-query-builder.ts

```diff
@@ -716,6 +716,66 @@ export class DeleteQueryBuilder<DB, TB extends keyof DB, O>
     })
   }
 
+  /**
+   * This can be used to add any additional SQL to the front of the query __after__ the `delete` keyword.
+   *
+   * ### Examples
+   *
+   * ```ts
+   * await db.deleteFrom('person')
+   * .modifyFront(sql.raw('-- This is a comment\n'))
+   * .where('first_name', '=', 'John')
+   * .execute()
```

Comment:
Is there a real-world use case for this?

Reply:
No, made just for consistency, will remove it